src/auth/awsrequestsigner.ts

```diff
@@ -60,15 +49,15 @@ interface GenerateAuthHeaderMapOptions {
   // The AWS service URL query string.
   canonicalQuerystring: string;
   // The HTTP method used to call this API.
-  method: HttpMethod;
+  method: string;
```

Comment:
why is this type changing? seems more flexible? also not clear if this change has to do with the gaxios upgrade.